docs/02.API-REFERENCE.md

```diff
@@ -10423,7 +10368,8 @@ jerry_generate_snapshot (const jerry_char_t *source_p,
   - thrown error, otherwise.
 
 *New in version 2.0*.
-*Changed in version [[NEXT_RELEASE]]*: The `resource_name_p`, and `resource_name_length` arguments are replaced by `options_p`.
+*Changed in version [[NEXT_RELEASE]]*: The `source_p`, `source_size`, `resource_name_p`,
```

Comment:
AFAIK there should be a newline before this. In addition the fact that a compiled/parsed JS code should be passed as first argument should also be mentioned here (as this is what changed since the previous versions).